.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -5,18 +5,21 @@ body:
   - type: markdown
     attributes:
       value: |
-        Before opening a new issue, take a moment to search through the current open ones.
+        ## Before opening a new issue, please take a moment to search through the current open and closed issues to check if it already exists.
 
         ---
 
   - type: dropdown
     id: type
     attributes:
-      label: Bug or Regression?
-      description: Is this a bug or a regression?
+      label: Regression?
+      description: |
+        Regression means that something used to work but no longer does.
+        **BEFORE CONTINUING**, please check if this bug is a regression or not, and if it is, we need you to bisect with the help of the wiki: https://wiki.hyprland.org/Crashes-and-Bugs/#bisecting-an-issue
+      multiple: true
```

Comment:
I don't think users should be able to select both Yes and No.

Reply:
That option achieves that there is no option selected by default, so it forces the user to select one option to be able to submit the bug report. That outweighs the possibility of one mad lad deciding to select both IMO.

Reply:
Then why not set `default: 0` instead? This will force the user to choose one of the provided options, according to the [GitHub docs](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema#validations-for-dropdown).

Reply:
What's the problem with selecting "yes" as a default? It can still be changed.

Reply:
The problem is that people have attention problems and just ignore it. If we don't have a default, people can't send the bug report unless they explicitly select yes or no, so you can expect that the value of regression is hand picked, and not just "yes" because the user ignored it.

Reply:
Alright then, makes sense.